profiles/serializers.py

```diff
@@ -39,17 +42,18 @@ def create(self, validated_data):
     def update(self, instance, validated_data):
         profile_data = validated_data.pop('profile', None)
 
+        instance.email = validated_data.pop('email', instance.email)
+        instance.save()
```

Comment:
Might be better to do this along with `profile.save` below just in case there's an exception in setting one of the fields

Reply:
Yeah, I'll wrap this in a transaction so it all either succeeds or fails.